### ISSUE TITLE
Show file sizes

### ISSUE THREAD

**Issue by phil294** [go-gitea/gitea#27724]
### Feature Description

This is a resurrection of #3953 which the author closed for reasons I don't understand.

"It would be nice to have the [file size] size in the [file listing] as well ([like this extension does it for GitHub](https://chrome.google.com/webstore/detail/github-repository-size/apnjnioapinblneaedefcnopcjepgkci))."

### Screenshots

![](https://lh3.googleusercontent.com/Dj3Y6TrPu07ue0Daos4lO-OgrGkmWefMZpPYxFPNnbVzFH_AZSHQlBIWpL0UCoVhvlKV4_C3dRQ1u0m5j8n58nUP=w640-h400-e365-rj-sc0x00ffffff)

**Comment by sgabenov** [comment]
I don't think that it is very useful to calculate size for source code text files. Mostly LFS can consume repo size, but to have a file in repo is not a big deal overall

**Comment by bauermarkus** [comment]
There are many cases where a repo is used not only for code, also for ressources and other things

It can be useful

**Comment by lunny** [comment]
I don't think we need that. And both GitLab and Github doesn't display it.

**Comment by wxiaoguang** [comment]
It seems that in most cases, Gitea would just follow GitHub/GitLab's design, so at the moment there is no need to add the "file size". Maybe the decision could be changed in the future, then let's do it at that time.
